packages/opentelemetry-exporter-zipkin/src/platform/browser/zipkin.ts

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as api from '@opentelemetry/api';
+import { ExportResult, NoopLogger } from '@opentelemetry/core';
+import { SpanExporter, ReadableSpan } from '@opentelemetry/tracing';
+import { SERVICE_RESOURCE } from '@opentelemetry/resources';
+import * as zipkinTypes from './../../types';
+import {
+  toZipkinSpan,
+  statusCodeTagName,
+  statusDescriptionTagName,
+} from './../../transform';
+import { OT_REQUEST_HEADER } from './../../utils';
+/**
+ * Zipkin Exporter
+ */
+export class ZipkinExporter implements SpanExporter {
```

Comment:
This is almost the same as the node version, but is completely copied. Is there no way to use a base class to share functionality?